resources/app/shared/includes/Header.vue

```diff
@@ -1,380 +1,830 @@
 <template>
-    <nav class="sticky top-0 h-32 flex flex-col w-full text-gray-100 bg-gray-900 border-t border-t-2 border-r border-theme-2 body-font shadow z-50">
+    <nav
+        class="sticky top-0 flex flex-col w-full text-gray-100 bg-gray-900 border-t border-t-2 border-r border-theme-2 body-font shadow z-50"
+    >
         <div class="w-full items-center">
             <div class="transition duration-500 ease-in-out transform">
-                <div class="flex flex-col flex-wrap p-2 md:items-center md:flex-row">
-                    <inertia-link :href="route('dashboard.index')" class="p-1 mx-2 border-transparent text-gray-400 rounded-full hover:text-white bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700">
-                        <svg class="h-5 w-5" stroke="currentColor" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
-                            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6" />
+                <div
+                    class="flex sm:flex-row flex-wrap p-2 md:items-center md:flex-row"
+                >
+                    <inertia-link
+                        :href="route('dashboard.index')"
+                        class="p-1 mx-2 border-transparent text-gray-400 rounded-full hover:text-white bg-gray-700 focus:outline-none focus:text-white focus:bg-gray-700"
+                    >
+                        <svg
+                            class="h-5 w-5"
+                            stroke="currentColor"
+                            xmlns="http://www.w3.org/2000/svg"
```

Comment:
Please use the `icon` component here for adding icons to the repo. We use https://heroicons.com for our reference.

Reply:
Sure, I am doing the required changes,